src/utils/web3.ts

```diff
@@ -31,6 +31,15 @@ export function accountTruncate(account: string): string {
   return truncated
 }
 
+export function nameTruncate(name: string): string {
```

Comment:
It would be nicer to have an optional maxLength param, and use that when calling this code.

Reply:
Field added